config_registry

```diff
@@ -38,9 +40,11 @@ cortexa15:   cortexa15/armv7a
 cortexa9:    cortexa9/armv7a
 
 # IBM architectures.
-power9:	     power9/generic
+power9:	     power9
```

Comment:
Is this part actually addressed by this PR? I would think this line should stay the same...